fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/debug/Debugger.java

```diff
@@ -919,11 +922,29 @@ public PEvent.Builder toEventBuilder() {
      * structures of the planner.
      */
     class InsertIntoMemoEvent implements Event {
+        @Nonnull
+        private final RelationalExpression expression;
+
         @Nonnull
         private final Location location;
 
-        public InsertIntoMemoEvent(@Nonnull final Location location) {
+        @Nonnull
+        private final List<Reference> reusedExpressionReferences;
+
+        public InsertIntoMemoEvent(@Nonnull final RelationalExpression expression,
```

Comment:
Nitpicking here and the other constructor. Can you please make the location come first?